src/types.ts

```diff
@@ -37,14 +37,14 @@ export type IOState = {
   records: Records; // The list of all ArNS names and their associated data
   gateways: Gateways; // each gateway uses its public arweave wallet address to identify it in the gateway registry
   fees: Fees; // starting list of all fees for purchasing ArNS names
-  settings: ContractSettings; // protocol settings and parameters
   reserved: ReservedNames; // list of all reserved names that are not allowed to be purchased at this time
   auctions: Auctions;
   lastTickedHeight: number; // periodicity management
   demandFactoring: DemandFactoringData;
   observations: Observations;
   distributions: EpochDistributionData;
   vaults: RegistryVaults;
+  settings?: unknown; // satisfies base PstContract type
```

Comment:
very odd

Reply:
`PstState` is a provided type from `warp`. We loosely follow it but removed it here bc i think we need to still decide if it's the format we want to follow.

Reply:
[73617ed](https://github.com/ar-io/testnet-contract/pull/184/commits/73617ed4fd96beb369fefd8da6cfcc09ba3e0d03)